src/v2/Apps/Artwork/Components/ArtworkSidebar/LotTimer.tsx

```diff
@@ -5,22 +5,58 @@ import { Text, Spacer, Box } from "@artsy/palette"
 import { useTimer } from "v2/Utils/Hooks/useTimer"
 import { getSaleOrLotTimerInfo } from "v2/Utils/getSaleOrLotTimerInfo"
 import { ArtworkSidebarAuctionProgressBar } from "./ArtworkSidebarAuctionProgressBar"
+import { useEffect } from "react"
+import { getENV } from "v2/Utils/getENV"
 
 export interface LotTimerProps {
   saleArtwork: LotTimer_saleArtwork
 }
 
 export const LotTimer: React.FC<LotTimerProps> = ({ saleArtwork }) => {
-  const { endAt, extendedBiddingEndAt } = saleArtwork
+  const { endAt, extendedBiddingEndAt, lotID } = saleArtwork
 
-  const startAt = saleArtwork?.sale?.startAt
+  const startAt = saleArtwork.sale?.startAt
   const extendedBiddingPeriodMinutes =
-    saleArtwork?.sale?.extendedBiddingPeriodMinutes
+    saleArtwork.sale?.extendedBiddingPeriodMinutes
   const extendedBiddingIntervalMinutes =
-    saleArtwork?.sale?.extendedBiddingIntervalMinutes
+    saleArtwork.sale?.extendedBiddingIntervalMinutes
 
   const biddingEndAt = extendedBiddingEndAt ?? endAt
-  const { hasEnded, time, hasStarted } = useTimer(biddingEndAt!, startAt!)
+
+  const [updatedBiddingEndAt, setUpdatedBiddingEndAt] = React.useState(
+    biddingEndAt
+  )
+  const [isExtended, setIsExtended] = React.useState(false)
+
+  useEffect(() => {
+    if (extendedBiddingIntervalMinutes) {
+      const actionCable = require("actioncable")
+      const CableApp = {} as any
+      CableApp.cable = actionCable.createConsumer(
+        getENV("GRAVITY_WEBSOCKET_URL")
+      )
+
+      CableApp.cable.subscriptions.create(
```

Comment:
It's incredibly cool that this is all it takes to setup websockets. I would have thought there was going to be way more overhead!